.github/workflows/ci.yml

```diff
@@ -18,15 +18,18 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v2
         with:
-          python-version: 2.7
+          python-version: 3.8
 
       - name: Install Python dependencies
         run: |
           pip install --upgrade pip
-          pip install yadage
+          pip install --pre reana-client
 
-      - name: Validate workflow
-        run: yadage-validate workflow/yadage/workflow.yaml | grep -q 'workflow validates'
+      - name: Validate workflows
+        run: |        
+          for file in $FILES; do reana-client validate -f $file; done
+        env:
+          FILES: reana.yaml reana-slurmcern.yaml reana-htcondorcern.yaml reana-yadage.yaml reana-yadage-slurmcern.yaml reana-yadage-htcondorcern.yaml reana-cwl.yaml reana-cwl-slurmcern.yaml reana-cwl-htcondorcern.yaml 
 
       - run: rm -rf results && mkdir results
 
```

Comment:
Should we add this and the two `run` below to a single step `Run workflows`?
And we can also add sections for other workflow engines as @tiborsimko mentioned here:

> If we want to get extra fancy, we could introduce separate "Run workflow" tests with running on naked CWL/Yadage/Serial-on-Docker/Snakemake environment, using the usual out-of-REANA instructions that we document there, i.e. mkdir snakemake-local-run etc.